app/Controller/UsersController.php

```diff
@@ -726,18 +726,8 @@ public function resetauthkey($id = null) {
 		$this->redirect($this->referer());
 	}
 
-	public function memberslist() {
-		// Orglist
-		$fields = array('Organisation.name', 'count(User.id) as `num_members`');
-		$params = array(
-				'fields' => $fields,
-				'recursive' => -1,
-				'contain' => array('Organisation'),
-				'group' => array('Organisation.name', 'Organisation.id'),
-				'order' => array('UPPER(Organisation.name)'),
-		);
-		$orgs = $this->User->find('all', $params);
-		$this->set('orgs', $orgs);
+	public function attributehistogram() {
+	    //all code is called via JS
```

Comment:
it's needed to generate the view. If you call a /attributehistogram in the URL but there is no function with the same name in the controller the page will not be displayed. 
MVC principle + CakePHP. (and actually other fw work the same way here)